en/developer_guide/bundle_configuration.rst

```diff
@@ -0,0 +1,178 @@
+| redirection                  | [class, repository] | The document and repository class        |
++------------------------------+---------------------+------------------------------------------+
+| keyword                      | [class, repository] | The document and repository class        |
++------------------------------+---------------------+------------------------------------------+
+| embed_keyword                | [class]             | The document class                       |
++------------------------------+---------------------+------------------------------------------+
+| translated_value             | [class]             | The document class                       |
++------------------------------+---------------------+------------------------------------------+
+
+MediaBundle
+~~~~~~~~~~~
+
++--------------------------------+---------------+-------------------------------------------------------------------------------------------------+
+| Key                            | Default value | Usage                                                                                           |
++================================+===============+=================================================================================================+
+| tmp_dir                        | /tmp          | The temporary dir to upload and resize the files                                                |
++--------------------------------+---------------+-------------------------------------------------------------------------------------------------+
+| filesystem                     | media_storage | The filesystem used to store the uploaded files                                                 |
++--------------------------------+---------------+-------------------------------------------------------------------------------------------------+
+| compression_quality            | 75            | The compression quality for the thumbnails                                                      |
++--------------------------------+---------------+-------------------------------------------------------------------------------------------------+
+| thumbnail.$key.max_height      | none          | A image format name by $key, the result thumbnail will fit in a box with max_height px          |
++--------------------------------+---------------+-------------------------------------------------------------------------------------------------+
+| thumbnail.$key.max_width       | none          | A image format name by $key, the result thumbnail will fit in a box with max_width px           |
++--------------------------------+---------------+-------------------------------------------------------------------------------------------------+
+| thumbnail.$key.[height, width] | [none, none]  | A image format name by $key, the result thumbnail will fit in a box with height px and width px |
```

Comment:
thumbnail.$key is not realy clear to
It represent 
thumbnail:
    $key:
         max_height: ...px
or 
thumbnail.$key.max_height: ...px